lib/utils/AbstractQueryConfiguration.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:openfoodfacts/utils/LanguageHelper.dart';
+import 'package:openfoodfacts/utils/ProductFields.dart';
+
+/// Abstract Query Configuration, that helps build API URI
+abstract class AbstractQueryConfiguration {
+  OpenFoodFactsLanguage? language;
+  // Allow apps to directly provide the language code and country code without
+  // having to use the OpenFoodFactsLanguage helper.
+  String? lc;
```

Comment:
Rather than using `lc` and `cc` should we spell out the names for clarity? Or is this convention very common?

Reply:
I won't address that subject here as 1) I just reproduced was there before, and more important 2) there are currently a lot of talking about languages in #185.

Reply:
Fair point 👍